jest.setup.js

```diff
@@ -0,0 +1,10 @@
+// TODO [#339]: replace third-party libraries with NPM modules
+import jQuery from './_dprhtml/js/external/jquery-3.4.1.min';
+import knockout from './_dprhtml/js/external/ajax/libs/knockout/3.5.0/knockout-min';
+
+window.jQuery = jQuery;
+window.ko = knockout;
+
+window.DPR_G = require('./_dprhtml/js/globalObject');
+window.DPR_PAL = require('./_dprhtml/js/dpr_pal');
+window.DPR_prefload_mod = require('./_dprhtml/js/prefload');
```

Comment:
I'm using `require` here rather than `import` because many DPR modules expect jQuery and Knockout to be defined globally when they are loaded. Since the module loader runs `import` statements before any other code, `window.jQuery` and `window.ko` will not be assigned in time when the DPR modules are loaded.